src/shell/shell_batch.cpp

```diff
@@ -138,6 +139,9 @@ bool BatchFile::ReadLine(char * line) {
 				next -= '0';
 				if (cmd->GetCount()<(unsigned int)next) continue;
 				std::string word;
+				assert(next >= 0);
+				if (!cmd->FindCommand(static_cast<unsigned>(next), word))
+					continue;
```

Comment:
Please mention in the commit message, that this commit brings back functionality accidentally removed in the previous commit.

Reply:
Thanks for catching this! Will fix it directly.